src/lucky/routable.cr

```diff
@@ -173,7 +174,7 @@ module Lucky::Routable
       ERROR
       %}
     {% else %}
-      {% NORMALIZED_ROUTES << {
+      {% NORMALIZED_ROUTES[normalized_key] = {
            normalized_path: normalized_path,
            original_path:   original_path,
            method:          method,
```

Comment:
My first thought was "we don't need most of this now". The `action` key is really the only value being used, but maybe it doesn't hurt to have the extra data?

Reply:
I was thinking the same thing, but I guess it's only present and compile time and not in the actual executable in any case? I also didn't want to change anything that wasn't completely necessary from the original implementation without first discussing it with the rest of you. 🙂